lib/ui/painting/single_frame_codec.h

```diff
@@ -31,6 +32,8 @@ class SingleFrameCodec : public Codec {
 
  private:
   ImageDecoder::ImageDescriptor descriptor_;
+  fml::RefPtr<FrameInfo> frame_;
```

Comment:
`cached_frame_` maybe to indicate that this is not something that is implicitly created?